openedx/core/djangoapps/theming/apps.py

```diff
@@ -0,0 +1,82 @@
+
+import os
+from django.apps import AppConfig
+from django.conf import settings
+from django.core.checks import Error, Tags, register
+
+
+class ThemingConfig(AppConfig):
+    name = 'openedx.core.djangoapps.theming'
+    verbose_name = "Theming"
+
+
+@register(Tags.compatibility)
+def check_comprehensive_theme_settings(app_configs, **kwargs):
```

Comment:
All the previous scattered checks of `COMPREHENSIVE_THEME_DIR` and `COMPREHENSIVE_THEME_DIRS` has been moved to this method.